README.md

```diff
@@ -103,8 +107,19 @@ for more detail on querying.
 
 ## Optional
 
-An alternative to using `curl` for ES config and querying is
-[kibana](https://www.elastic.co/products/kibana)
+An alternative to using `curl` for ES config, querying and also for visualisation is [kibana](https://www.elastic.co/products/kibana).
+For the latest version use `brew install kibana`. Similarly you can use `apt-get` or `yum` for other platforms.
+
+Troubleshooting:
+* In case you have any issues of compatibility, please check [here](https://www.elastic.co/support/matrix#show_compatibility).
+
+Quickstart: 
+* On first use please remember the default port for ES is `9200` and the default index is `profiles`. [Here](https://www.youtube.com/watch?v=mMhnGjp8oOI) is a comprehensive intro.
+* The examples for the `Dev Tools` tab are the same as the ones using `curl`.
+* Example query for the `Discover` tab when you choose the `"name", "alternativeName", "address.addressLines", "address.postcode", "doctors"` as the source:
```

Comment:
The previous comment about this not making sense still stands. I just don't see what value it adds, nor the query below.